src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -538,21 +538,26 @@ private GenotypesContext mergeRefConfidenceGenotypes(final VariantContext vc,
             final int ploidy = g.getPloidy();
             final GenotypeBuilder genotypeBuilder = new GenotypeBuilder(g);
             if (!doSomaticMerge) {
-                if (g.hasPL()) {
-                    // lazy initialization of the genotype index map by ploidy.
+                if (g.hasPL() | g.hasAD()) {
```

Comment:
Although technically the bitwise or operator `|` does what you want here, convention is to use conditional-or `||` unless you really want the non-conditional behavior.